Add files via upload

Daniel1815 opened the pull request:
How do I make the chart fit itself to whatever screen it's shown on? As is, both of them require scroll bars, and when I save that image, it only shows the part of the chart that is on the screen, it doesn't expand the chart so as to no longer need the bars

Daniel1815 committed:
Add files via upload

How do I make the chart fit itself to whatever screen it's shown on? As is, both of them require scroll bars, and when I save that image, it only shows the part of the chart that is on the screen, it doesn't expand the chart so as to no longer need the bars

Daniel1815 committed:
Add files via upload

First draft of Analysis project

whanley commented:
I have posted instructions to embed charts here: https://dig-eg-gaz.github.io/how-to/serial-question-instructions/#how-do-i-show-these-visualizations-in-my-markdown-page

whanley commented on DanielArenas-Analysis.md:
```diff
@@ -0,0 +1,24 @@
+title: Attitudes towards Germany: Random, Situational, or Seasonal?
+author: DanielArenas
+---
+1905 was a watershed year in the antebellum period preceding WWI. Tensions ran high as Russia was in the process of earning the distinct honor of being the first European nation to lose a war to a non-Western power (Japan), and the great powers were concerned if this, and the nascent but ultimately futile 1905 revolution, meant the decline of Russia as a major imperial power, which would certainl ...
```
- antebellum preceding WWI is redundant.
Good intro

whanley replied:
In markdown, you need two line breaks between each paragraph.

whanley commented on DanielArenas-Analysis.md:
```diff
@@ -0,0 +1,24 @@
+---
+1905 was a watershed year in the antebellum period preceding WWI. Tensions ran high as Russia was in the process of earning the distinct honor of being the first European nation to lose a war to a non-Western power (Japan), and the great powers were concerned if this, and the nascent but ultimately futile 1905 revolution, meant the decline of Russia as a major imperial power, which would certainl ...
+	As the main English newspaper in colonial Egypt, the Gazette should roughly represent the overall average of British views, but what surprised me the most working on my week was the sheer variety of opinions, and none felt so jarringly dissimilar as those concerning Germany. With this project, I sought to find out whether there was a discernable pattern in when the Gazette published editorial ar ...
+	Methodology and Structure
```
Add a # before headings

whanley commented on DanielArenas-Analysis.md:
```diff
@@ -0,0 +1,24 @@
+	This investigation sought to do two things; quantify a subjective piece of data (occurrence of a pro- or anti- Germany article), collect the data, and observe whether any patterns emerged.
+The biggest potential fault of this investigation comes from the very beginning: how to measure what is an article’s stance on Germany? From the start I wanted to look at one thing in particular, not wires/telegrams, but articles. These were difficult to isolate because in the newspaper itself there is little regularity in which these appear and how they are presented, which means that in the enc ...
+	To do this, I used the following xpath queries:
+//div[@type=”article”]//p[contains(.,”Germany”)]
```
put code between two ` ` marks

whanley replied:
```

whanley replied:
Um I mean the mark under the ~

whanley reviewed (commented):
Good. Maybe do something about word frequency in positive v. negative articles?
- clean up markdown and embed tableau workbooks

Daniel1815 committed:
Add files via upload

I have made all the changes you asked for in your comments. I am not sure if the embed works, please let me know if it doesnt

Daniel1815 committed:
Add files via upload

Daniel1815 committed:
Delete DanielArenasChartComparisonByFrequency.twb

Daniel1815 committed:
Delete DanielArenasChartComparisonByType.twb

Daniel1815 commented:
deleted the two tableau files and edited the main analysis project.
